src/Pages/CartList/CartList.js

```diff
@@ -0,0 +1,197 @@
+import React from 'react';
+import Header from '../../Component/HeaderComponent/Header';
+import Footer from '../../Component/FooterComponent/Footer';
+import CartItem from '../CartList/CartItem';
+import './CartList.scss';
+
+class CartList extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      cartData: [],
+      selectedArr: [],
+    };
+  }
+
+  componentDidMount() {
+    fetch('/Data/CartData.json')
+      .then(response => response.json())
+      .then(data => {
+        this.setState({ cartData: data });
+      });
+  }
+
+  handleQuantity = event => {
+    const { cartData } = this.state;
+    const { id, className } = event.target;
+    if (className === 'quantity-plus') {
+      const newQuantity = cartData.map((cartItem, index) => {
+        return parseInt(id) !== index + 1
+          ? cartItem
+          : {
+              ...cartItem,
+              quantity: cartItem.quantity + 1,
+            };
+      });
+      this.setState({ cartData: newQuantity });
+    } else if (className === 'quantity-minus') {
+      const newQuantity = cartData.map((cartItem, index) => {
+        if (parseInt(id) !== index + 1) {
+          return cartItem;
+        } else {
+          return {
+            ...cartItem,
+            quantity:
+              cartItem.quantity > 1 ? cartItem.quantity - 1 : cartItem.quantity,
+          };
+        }
```

Comment:
- 얼리 리턴 패턴 알려드립니다! 참고해보세요!
```suggestion
    if (
      className === 'quantity-minus' &&
      cartData[Number(id - 1)].quantity === 1
    )
      return;

    const newQuantity = cartData.map((cartItem, index) => {
      return parseInt(id) !== index + 1
        ? cartItem
        : {
            ...cartItem,
            quantity:
              className === 'quantity-plus'
                ? cartItem.quantity + 1
                : cartItem.quantity - 1,
          };
    });
    this.setState({ cartData: newQuantity });
```

Reply:
참고가 되었습니다! 감사합니다👍